pkg/metrics/metrics.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// ServerMetrics contains Prometheus metrics for the Ark server.
+type ServerMetrics struct {
+	metrics map[string]prometheus.Collector
+}
+
+const (
+	metricNamespace             = "ark_server"
+	backupTarballSizeBytesGauge = "backup_tarball_size_bytes"
+	backupAttemptsCount         = "backup_attempts_total"
```

Comment:
I would like to be consistent with singular vs plural and our nomenclature. Should this be attempt or attempts?

success or successes?

failure or failures?